docs/tutorial/en/source/tutorial/MCP.py

```diff
@@ -0,0 +1,149 @@
+        "puppeteer": {
+            "command": "npx",
+            "args": ["-y", "@modelcontextprotocol/server-puppeteer"],
+        },
+    },
+}
+
+# %%
+# Initialize ServiceToolkit and add MCP server configuration (uncomment the
+# code lines below)
+
+toolkit = ServiceToolkit()
+# toolkit.add_mcp_servers(server_configs=local_configs)
+
+# %%
+# Output tool configuration instructions to verify settings
+print(toolkit.tools_instruction)
+
+# %%
+# If you need to use the HTTP protocol to connect to a remote server (or an MCP server running locally in another process using the SSE protocol), you can refer to the following configuration for the connection.
+
+remote_configs = {
+    "mcpServers": {
+        "puppeteer": {
+            "url": "http://127.0.0.1:8000/sse",
+        },
+    },
+}
+
+# Recycle the toolkit instance
+del toolkit
+
+# %%
+# Adding multi-agent applications created with AgentScope to your own MCP server
+# --------------------------
+# You can easily add multi-agent applications to your own MCP server using the official MCP Python SDK. The following example code demonstrates how to achieve this purpose using AgentScope and the MCP library.
+
+import agentscope
+
+from agentscope.agents import DialogAgent
+from agentscope.message import Msg
+from mcp.server import FastMCP
+from pydantic import Field
+
+agentscope.init(
+    model_configs={
+        "config_name": "my-qwen-max",
+        "model_type": "dashscope_chat",
+        "model_name": "qwen-max",
+    },
+)
+
+mcp = FastMCP("My MCP Server")
+
+
+# %%
+# Use the @mcp.tool() decorator to define tools; here we create a joke-telling tool using AgentScope's `DialogAgent`.
+
+
+@mcp.tool()
+def tell_a_joke(
+    topic: str = Field(
+        description="The topic of a joke",
+    ),
+) -> int:
+    """Generate a joke based on the given topic"""
+    agent = DialogAgent(
+        name="FunnyBot",
+        model_config_name="my-qwen-max",
+        sys_prompt="You are a witty comedian tasked with creating a joke about the given topic.",
+    )
+    msg_task = Msg("user", topic, "user")
+    res = agent(msg_task)
+
+    return res.content
+
+
+# %%
+# Save the above code to the file `my_mcp_server.py`.
+# Then, run the following command in the terminal to start the server:
+# `mcp run my_mcp_server.py -t sse`
+# This command starts the MCP server and transmits the results of tool calls via Server-Sent Events (SSE).
+# This way, you can access and use the multi-agent application through the configured MCP server.
+
+# %%
+# Principle of AgentScope integrating MCP
+# --------------------------
+# Since the client interface provided by the MCP Python SDK is asynchronous, the developers of AgentScope have encapsulated synchronous interfaces for user convenience.
+# Through this encapsulation, users can call asynchronous methods in a synchronous environment, thus improving the usability and readability of the code.
+
+# %%
+from agentscope.service.mcp_manager import sync_exec
```

Comment:
1. from agentscope.service import sync_exec
2. Should we expose this function to developers or not, and will they use this function?